consoles/sshVirtsh.pm

```diff
@@ -34,7 +36,9 @@ sub new {
     $self->instance(get_var('VIRSH_INSTANCE', 1));
     # default name
     $self->name("openQA-SUT-" . $self->instance);
-    $self->_init_xml;
+    $self->vmm_family(get_var('VIRSH_VMM_FAMILY'));
+    $self->vmm_type(get_var('VIRSH_VMM_TYPE'));
```

Comment:
I assume kvm should be the default, right? Please use
```perl
$self->vmm_family(get_var('VIRSH_VMM_FAMILY', 'kvm'));
$self->vmm_type(get_var('VIRSH_VMM_TYPE', 'hvm'));
```
Or are those two vars supposed to be mandatory? Then I'm missing check for presence.